Api

pablo-grande opened the pull request.

Commit:
Closes #4

Pic field non-mandatory

Commit:
Closes #5

Check for name in string, if not put id, breed and species

Nosk14 commented:
https://github.com/petconnection/backend/blob/03cdb2297a4bc84134f67386df3701323e1f331d/api/models.py#L44-L49

I really like the string representation when an animal has not a name. Now, since the name field is not unique, I wonder if the string representations should always be the one without the name.
What do you think?

pablo-grande commented:
We can make a mix: if there is a name, put it, if not, put the id; but in both cases put breed and species.
I believe there won't be 2 animals named the same in the same shelter, at least of the same species.

Commit:
Changing naming convention for animal

Commit:
Add MedicalRecords as OneToOneRelationship

A MedicalRecord will have an even closer relationships with its animal.
This means they share same id (animal.id == medicalrecord.id),
furthermore the `<animal>.medicalrecord` property is automatically added
